### PR TITLE
Traffic sign detection

### LINKED ISSUE #189: Traffic Signs Recognition using CNN and Keras
To recognize the traffic signs i want to use CNN which consists of 4 layers that extracts features very well and provides us with an accuracy recognition.
please assign this to me ..

### PR DESCRIPTION
**Pull request by ramana2074**
## Related Issue
Traffic Sign Detection
NOTE: Kindly write in the following format -

Closes #189 

## Description
Using German Traffic Sign Recognition Benchmark, in this script I have used
Data Augmentation and then passed the images to miniVGGNet (VGG 7) a famous architecture of
Convolutional Neural Network (CNN).

## Type of PR

- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Checklist:
- [X] I have performed a self-review of my code
- [X] I have read and followed the Contribution Guidelines.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Aryan-Chharia** [comment]
Add results

**Comment by ramana2074** [comment]
Results are already in ipynb files is it not enough?
if you want it i will add them

**Comment by Aryan-Chharia** [comment]
add them in a seperate folder showing the input and output
